NDTensors/src/abstractarray/set_types.jl

```diff
@@ -37,30 +37,34 @@ function set_indstype(arraytype::Type{<:AbstractArray}, dims::Tuple)
   return set_ndims(arraytype, length(dims))
 end
 
-function set_eltype_if_unspecified(
+function specify_eltype(
```

Comment:
I was picturing that we would use `SetParameters.specify_parameters` for this (currently `SetParameters.set_unspecified_parameters` but needs to be renamed), i.e.:
```julia
function specify_eltype(  arraytype::Type{<:AbstractArray}, eltype::Type=default_eltype())
  return specify_parameters(arraytype, Position(1), eltype)
end
```
Is that the plan here?